### PR TITLE
Navigation collapse

### PR DESCRIPTION
**Pull request by gary149**
It's more of a Zen mode (compared to other chat apps):
- It doesn't persist on page reload.
- You cannot create a new conversation without expanding the menu.

https://github.com/huggingface/chat-ui/assets/3841370/a6768879-4744-4276-959b-e2792df381df

### CODE REVIEW

**Review by mishig25 — APPROVED** [review]
lgtm !

File: src/lib/components/ExpandNavigation.svelte [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<script lang="ts">
+	export let isCollapsed: boolean;
+	export let classNames: string;
+</script>
+
+<button
+	on:click
+	class="{classNames} group absolute flex h-16 w-6 flex-col items-center justify-center -space-y-1 outline-none *:h-3 *:w-1 *:rounded-full *:hover:bg-gray-300 max-md:hidden dark:*:hover:bg-gray-600 {!isCollapsed
```
Comment (mishig25) [inline src/lib/components/ExpandNavigation.svelte:8]: `absolute` is already declared in https://github.com/huggingface/chat-ui/blob/navigation-collapse/src/routes/%2Blayout.svelte#L188

Comment (mishig25) [inline src/lib/components/ExpandNavigation.svelte:8]:
```suggestion
	class="{classNames} group flex h-16 w-6 flex-col items-center justify-center -space-y-1 outline-none *:h-3 *:w-1 *:rounded-full *:hover:bg-gray-300 max-md:hidden dark:*:hover:bg-gray-600 {!isCollapsed
```

`absolute` is already declared in https://github.com/huggingface/chat-ui/blob/navigation-collapse/src/routes/%2Blayout.svelte#L188

**Review by nsarrazin — APPROVED** [review]
Very nice to have!

**Pull request merged**